source/index.html.md.erb

```diff
@@ -22,10 +22,16 @@ application will become available over the API.
 Providers can then use the API for:
 
 - reviewing applications
-- retrieving candidate application data
-- making (or amending) offers to candidates
-- confirming a candidate's place on a course
+  - [/retrieve-a_single-application](/retrieve-a_single-application)
+  - [/retrieve-many-applications](/retrieve-many-applications)
+- making, or amending, offers to candidates
+  - [/make-an-offer](/make-an-offer)
+  - [/amend-an-offer](/amend-an-offer)
+- confirming a candidates has met conditions, or has been enroled (onboarded)
```

Comment:
Typo: enrolled (but of course "enrolment" is the noun 🙃). Maybe remove "onboarded"?

Reply:
`onbordated` removed.